Lib/test/test_fractions.py

```diff
@@ -1220,6 +1220,30 @@ def test_invalid_formats(self):
                 with self.assertRaises(ValueError):
                     format(fraction, spec)
 
+    @requires_IEEE_754
+    def test_float_format_testfile(self):
+        from test.test_float import format_testfile
```

Comment:
I would prefer to just copy/paste the code to get the path here:

```py
format_testfile = os.path.join(test_dir, 'formatfloat_testcases.txt')
```

I prefer to not make tests inter-dependents. Sometimes, just importing a test has side effects.